tests/client/test_client_key_meilisearch.py

```diff
@@ -66,6 +68,7 @@ def test_update_keys(client):
     assert update_key['key'] is not None
     assert update_key['expiresAt'] is None
     assert update_key['actions'] == ['search']
+    client.delete_key(update_key['key'])
 
 def test_delete_key(client):
```

Comment:
Continuing with the fixture example above this would be:

```py
def test_delete_key(client, test_key):
    resp = client.delete_key(test_key['key'])
    assert resp.status_code == 204
```